pkg/tfutils/tfConfig.go

```diff
@@ -219,6 +197,40 @@ func ConfigureProvider(level string) {
 
 }
 
+func validateCfApiUrl(apiUrl string) {
+	if len(strings.TrimSpace(apiUrl)) == 0 {
+		cleanup()
+		fmt.Print("\r\n")
+		log.Fatalf("cf api URL not set. set CF_API_URL environment variable to set CF API endpoint")
+	}
+}
+
+func validateCfAuthenticationData(username string, password string, cfAccessToken string, cfRefreshToken string, cfClientId string, cfClientSecret string) {
+	if (len(strings.TrimSpace(username)) == 0 && len(strings.TrimSpace(password)) == 0) && (len(strings.TrimSpace(cfAccessToken)) == 0 && len(strings.TrimSpace(cfRefreshToken)) == 0) && (len(strings.TrimSpace(cfClientId)) == 0 && len(strings.TrimSpace(cfClientSecret)) == 0) {
```

Comment:
Since we do this zero-length string check for multiple arguments, across multiple functions, could we write a generic function that takes var args and returns true if the total length of all arguments combined is 0?